controllers/upgrade.go

```diff
@@ -112,7 +117,14 @@ func (r *RollingUpgradeContext) RotateNodes() error {
 		return nil
 	}
 
-	rotationTargets := r.SelectTargets(scalingGroup)
+	rotationTargets := r.SelectTargets(scalingGroup, failedDrainInstances)
+
+	if len(rotationTargets) == 0 && len(failedDrainInstances) > 0 {
+		// If there are failed instances, but no rotation targets, then we need to requeue.
```

Comment:
don't quite understand the comment here.

based on the logic, it should be:
```suggestion
		// If there are failed instances, but no rotation targets, then select failed isntances anyway.
```

Reply:
I agree, let me rephrase it

Reply:
fixed in [4511ff5](https://github.com/keikoproj/upgrade-manager/pull/394/commits/4511ff524f3093469ef17f72515d6d26f946b46c)